handlers/chat.js

```diff
@@ -279,7 +279,7 @@ var onOtherJoinedChannel = function onOtherJoinedChannel(message) {
     /* Someone joined a chat channel you're in. */
     var otherJoined = Dota2.schema.lookupType("CMsgDOTAOtherJoinedChatChannel").decode(message);
     var channel = this._getChannelById(otherJoined.channel_id);
-    this.Logger.debug(otherJoined.steam_id + " joined channel " + channel.channel_name);
+    this.Logger.debug(otherJoined.steam_id + " joined channel " + channel ? channel.channel_name : " unknown");
     this.emit("chatJoin",
         channel.channel_name,
```

Comment:
It would throw the same error here, channel would be undefined

Reply:
You're absolutely right, thanks.